src/Uno.UI/UI/Xaml/Shapes/Rectangle.iOSmacOS.cs

```diff
@@ -1,188 +1,55 @@
 ﻿using System;
-using System.Collections.Generic;
-using System.Text;
-using Windows.UI.Xaml.Media;
-using Uno.Extensions;
 using System.Linq;
-using Uno.Disposables;
 using Uno.UI.Extensions;
-using Uno.UI;
-using Foundation;
 using Windows.Foundation;
+using Windows.UI.Xaml.Media;
+using CoreGraphics;
 
-#if XAMARIN_IOS_UNIFIED
+#if __IOS__
 using UIKit;
-using CoreAnimation;
-using CoreGraphics;
-using _Color = UIKit.UIColor;
 using _BezierPath = UIKit.UIBezierPath;
 #elif __MACOS__
 using AppKit;
-using CoreAnimation;
-using CoreGraphics;
-using _Color = AppKit.NSColor;
 using _BezierPath = AppKit.NSBezierPath;
 #endif
 
 namespace Windows.UI.Xaml.Shapes
 {
 	public partial class Rectangle
 	{
-		CAShapeLayer _rectangleLayer = new CAShapeLayer();
-		CAGradientLayer _gradientLayer;
-		SerialDisposable _fillSubscription = new SerialDisposable();
-		SerialDisposable _strokeSubscription = new SerialDisposable();
-
 		public Rectangle()
 		{
-			//Set default stretch value
-			this.Stretch = Stretch.Fill;
-
-#if __IOS__
-			// Background color is black by default, if and only if overriding Draw(CGRect rect).
-			base.BackgroundColor = SolidColorBrushHelper.Transparent.Color;
-#else
-			base.WantsLayer = true;
-			base.Layer.BackgroundColor = SolidColorBrushHelper.Transparent.Color;
-#endif
```

Comment:
Where did this go ?

Reply:
In the [base class `Shape`](https://github.com/unoplatform/uno/pull/2911/files/#diff-1b16792b910be7fdd3f4ca3219e6d7c2R26) (ok I might have forgotten it and just pushed a commit to restore it ...)